app/controllers/quiz_controller.rb

```diff
@@ -27,14 +28,12 @@ def task
         end
       end
     when 6
-      answer = $data_67[params["question"].del_punc.chars.sort]
+      answer = $data_67[params["question"].del_punc.chars.sort].gsub(" —","")
     when 7
-      answer = $data_67[params["question"].del_punc.chars.sort]
+      answer = $data_67[params["question"].del_punc.chars.sort].gsub(" —","")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.